src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1212,7 +1221,7 @@
                :order-by [[time-field :desc]]
                :limit 1}]
     (sql/with-query-results result-set (hcore/format query)
-      (time-field (first result-set)))))
+      (some-> result-set first time-field))))
```

Comment:
... I can't remember. I can imagine it being relevant if there are no results, but 'first' and keyword-as-fn both pass through nils. I'll change it back.